### ISSUE TITLE
Bicycle mode does not work properly

### ISSUE THREAD

**Issue by jailby** [CUTR-at-USF/usf-mobullity#244]
When a use select the mode "Bicycle Only" or "Bus & Bicycle" the indicator in the itinerary returned is the  pedestrian indicator and the path is gray:
![image](https://cloud.githubusercontent.com/assets/1370401/14113748/492e3682-f5a2-11e5-941f-053fca3a42c6.png)

The bicycle mode is the same as walking.

It should be the same than for the mode rented bicycle:
![image](https://cloud.githubusercontent.com/assets/1370401/14113885/d5de3a96-f5a2-11e5-9d30-e4191c2772bc.png)

The path is colored purple (I think) and there is an indicator in the itenerary returned widget.

This issue is related to #242.

**Comment by jmfield2** [comment]
So, a partial fix for this is to remove the "Bicycle only" option since it doesn't currently do anything and the bike rental routing is triggered via the "_RENT" modifier on the "Rented bicycle" mode.

The "Bus & Bicycle" problem seems a bit more involved -  Just changing it to TRANSIT,BICYCLE_RENT does not fix the problem.. I either get all walking, or walk+bus itineraries.  At least for rentals, it seems like it would be difficult to get a route including bus and bike the way the bikeshare is currently setup.

**Comment by jmfield2** [comment]
So, I tried the tag closest to our master version (0.12.0) released back in November 2014, and was surprised to find the bike only mode working.  Although the diff was still pretty big, I diff the entire src/main and searched for "BICYCLE" and found some comments that stood out:

https://github.com/opentripplanner/OpenTripPlanner/blob/master/src/main/java/org/opentripplanner/api/parameter/QualifiedModeSet.java#L51

I changed the ONE line in my dev branch, and bike only + bus and bike works.  Good call to check earlier tags again! :)  

There are some other notable changes that we probably want to include at some point for "bike parking" but so far the routing seems to work fine.
